### PR TITLE
Sorting option for tags

### PR DESCRIPTION
**Pull request by rakeshPRaghu**
This PR will add an util function which can sort the tags in two ways, and Openebs changed to OpenEBS in some articles

1. Sorted by length:
    image
2. Sorted by name:
    image

Signed-off-by: Rakesh PR <rakesh.pr@mayadata.io>

### CODE REVIEW

File: website/src/utils/getTags.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
+ * 
+ * @param tagsDistribution contains the tag list with number of occurances
```
Comment (Pallavi-PH) [inline website/src/utils/getTags.ts:3]:
```suggestion
 * @param tagsDistribution contains the tag list with number of occurrences
```

File: website/src/components/MiniBlog/index.tsx [hunk 2]
```diff
@@ -128,7 +129,8 @@ const MiniBlog: React.FC = () => {
     }
   }, [jsonMdData]);
 
-  const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => {
+  const sortedTags = getTagsSorted(tagsDistribution,"length");
```
Comment (Pallavi-PH) [inline website/src/components/MiniBlog/index.tsx:132]: This sorting also needs to be by name

File: website/src/pages/Blog/index.tsx [hunk 3]
```diff
@@ -160,8 +161,8 @@ const Blog: React.FC = () => {
     />
     );
   };
-
-  const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => {
+  const sortedTags = getTagsSorted(tagsDistribution,"length");
```
Comment (Pallavi-PH) [inline website/src/pages/Blog/index.tsx:164]: This sorting also needs to be by name

Comment (Pallavi-PH) [reply website/src/pages/Blog/index.tsx:164]: Screenshot 2021-07-15 at 5 03 21 PM

**Comment by IsAmrish** [comment]
also @rakeshPRaghu , 
can we change `Cas` => `CAS`?

**Review by AVRahul — APPROVED** [review]

**Comment by Pallavi-PH** [comment]
Can you please change `OpenSource` to `Open Source` in tags? @rakeshPRaghu

**Review by IsAmrish — APPROVED** [review]
LGTM

**Review by Pallavi-PH — APPROVED** [review]
LGTM

File: website/src/utils/getTags.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+ * 
+ * @param tagsDistribution contains the tag list with number of occurrences
+ * @param sortBy can have two values name | count; default set as name
+ * @returns tags
+ * 
+ * ex : getTagsSorted(tagsDistribution,"count");
+ * for this function call the tags will be returned in sorted by count order
+ */
+
+const getTagsSorted = (tagsDistribution: any, sortBy: "name" | "count" = "name") => {
```
Comment (satyapriyamishra222) [inline website/src/utils/getTags.ts:11]: Should we rename this file to sortTags.ts  to justify its functionality ?

Comment (satyapriyamishra222) [reply website/src/utils/getTags.ts:11]: tagsDistribution: any, sortBy: "name" | "count" = "name" 
Does this has utility as we are always  sorting alphabetically.

File: website/src/utils/getTags.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  let tags = tagsDistribution;
+  tags =
+    sortBy === "name"
+      ? Object.keys(tagsDistribution).sort(function (a, b) {
+          return a.toLowerCase().localeCompare(b.toLowerCase());
+        })
+      : Object.keys(tagsDistribution).sort(function (a, b) {
```
Comment (satyapriyamishra222) [inline website/src/utils/getTags.ts:18]: Do we need this block ?

File: website/src/utils/getTags.ts (hunk 4 above)
Comment (rakeshPRaghu) [reply website/src/utils/getTags.ts:11]: i think we can keep this code, if we want to change the tag sort later

File: website/src/utils/getTags.ts (hunk 5 above)
Comment (rakeshPRaghu) [reply website/src/utils/getTags.ts:18]: if we want to show the tags wrt count order we can keep it, else we can remove the code.

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**